engine/consensus/approvals/tracker/tracker.go

```diff
@@ -170,7 +171,24 @@ func (st *SealingObservation) Complete() {
 
 	// dump observation to Logger
 	observation = observation.Int64("duration_ms", time.Since(st.startTime).Milliseconds())
-	observation.Msg("sealing observation")
+	reason := st.UnsealedReason()
+	observation.Msgf("sealing observation, unsealed reason: %v", reason)
+}
+
+func (st *SealingObservation) UnsealedReason() string {
+	if st.finalizedBlock == st.latestSealedBlock {
+		return "all finalized blocks have been sealed"
+	}
+
+	unsealedHeight := st.latestSealedBlock.Height + 1
+	if len(st.records) == 0 {
+		return fmt.Sprintf("no result found for next unsealed block at height: %v", unsealedHeight)
+	}
+
+	return fmt.Sprintf("unsealed block at height %v has been executed, the result has been incorporated on %v fork(s), "+
+		"but no result has received enough approvals, check the chunks_with_insufficient_approvals field for more details",
+		len(st.records),
```

Comment:
⚠️ 
`unsealed block at height %v` is filled with parameter `len(st.records)`

Reply:
⚠️ 
`result has been incorporated on %v fork(s)` This statement is only valid if there is one result. But we can have execution forks. Tow results might be incorporated in the _same_ fork